packages/experimental/Shadow/src/Shadow.tsx

```diff
@@ -6,21 +6,25 @@ import { getShadowTokenStyleSet } from './shadowStyle';
 
 export const Shadow = stagedComponent((props: ShadowProps) => {
   return (final: ShadowProps, children: React.ReactNode) => {
-    const shadowTokenStyleSet = getShadowTokenStyleSet(props.shadowToken);
-    const mergedProps = mergeProps(final, { style: shadowTokenStyleSet.ambient });
+    if (!props.shadowToken) {
+      return <>{children}</>;
+    } else {
```

Comment:
nit: We don't need to wrap the rest of the component in an else block. I generally prefer early returns over wrapping most of the code in an else block as it minimizes the "depth" of the code.